iml-api/src/graphql.rs

```diff
@@ -141,6 +142,86 @@ impl Deref for SortDir {
     }
 }
 
+#[derive(GraphQLEnum, sqlx::Type)]
+#[repr(i16)]
+enum MessageClass {
+    Normal = 0,
+    Lustre = 1,
+    LustreError = 2,
+    Copytool = 3,
+    CopytoolError = 4,
+}
+
+impl From<i16> for MessageClass {
+    fn from(value: i16) -> Self {
+        match value {
+            0 => MessageClass::Normal,
+            1 => MessageClass::Lustre,
+            2 => MessageClass::LustreError,
+            3 => MessageClass::Copytool,
+            4 => MessageClass::CopytoolError,
+            _ => panic!("Invalid variant"),
+        }
+    }
+}
+
+#[derive(GraphQLEnum, sqlx::Type)]
+#[repr(i16)]
+enum LogSeverity {
```

Comment:
Given these are sequential, enumerating the i16 for each one isn't necessary.

Reply:
I've specifically enumerated them since they are enumerated in Python and have to match.